src/pages/~/new.ts

```diff
@@ -1,10 +0,0 @@
-import type { APIRoute } from 'astro'
```

Comment:
My suggestion is to leave the existing route and others may have integrated w/ it.  Add yours in an additive/non-conflicting manner.